File: Retos/Reto #0/go/jorge-bizarro.go
package main

import (
	"fmt"
)

func main() {
	list := make([]int8, 100)

	for i := range list {
		valueNumber := i + 1
		valueString := ""

		if valueNumber%3 == 0 {
			valueString += "Fizz"
		}

		if valueNumber%5 == 0 {
			valueString += "Buzz"
		}

		if valueString == "" {
			fmt.Println(valueNumber)
		} else {
			fmt.Println(valueString)
		}
	}
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c++/jorge-bizarro.cpp
#include <iostream>

using std::cout;
using std::endl;

int main() {
  for (int i = 1; i < 101; i++) {
    if (i % 15 == 0) {
      cout << "FizzBuzz" << endl;
    } else if (i % 5 == 0) {
      cout << "Buzz" << endl;
    } else if (i % 3 == 0) {
      cout << "Fizz" << endl;
    } else {
      cout << i << endl;
    }
  }

  return 0;
}
